### PR TITLE
Apply patch to ignore rt_sigaction error if it is for SIGRTMAX when b…

### PR DESCRIPTION
**Pull request by nghiant2710**
…uilding golang images

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -28,10 +32,17 @@ case "$ARCH" in
...
 	;;
 esac
+
 # compile Go
 echo $GOARM
-cd go/src \
-	&& git checkout go$GOLANG_VERSION \
+cd go && git checkout COMMIT_$GOLANG_VERSION
+
+if version_cmp "1.5.2" $GOLANG_VERSION; then
```
Comment (petrosagg) [inline build.sh]: Does this mean it will only patch 1.5.2?

Comment (nghiant2710) [inline build.sh]: it means versions below 1.5.2

File: build.sh [hunk 2]
```diff
@@ -28,10 +32,17 @@ case "$ARCH" in
...
 esac
+
 # compile Go
 echo $GOARM
-cd go/src \
-	&& git checkout go$GOLANG_VERSION \
+cd go && git checkout COMMIT_$GOLANG_VERSION
+
+if version_le "1.5.2" $GOLANG_VERSION; then
+	cp /patches/golang-$GOLANG_VERSION.patch /go/golang-$GOLANG_VERSION.patch
```
Comment (petrosagg) [inline build.sh]: Is this needed? I think you can do `patch -p1 < /patches/golang-$GOLANG_VERSION.patch` directly

Comment (nghiant2710) [inline build.sh]: Updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**